apps/picsa-tools/option-tool/src/app/components/editor/editor.component.html

```diff
@@ -1,198 +1,208 @@
-                <input [(ngModel)]="practiceEntry" class="form-control">
-              </div>
-            </div> 
-          <div>
-            <button 
-            [style.opacity]="practiceEntry ? 1 : 0.5" 
-            [disabled]="!practiceEntry"  
-             style="background-color: #ccc;" mat-button
-            mat-button 
-            matStepperNext>Next</button>
+<div class="dialog-content">
+  <div class="warningText">{{warningText}}</div>
+  <button *ngIf="editMode" class="saveButtonPosition" mat-button mat-raised-button (click)="submitForm()"
+    color="primary">Save Changes</button>
+  <mat-stepper [linear]="isLinear" #stepper>
+    <mat-step matStepContent>
+      <ng-template matStepLabel>Practice</ng-template>
+      <div class="StepContainer">
+        <div class="StepTitle">Enter the practice for this option</div>
+        <div class="StepContent">
+          <div class="form-group">
+            <label>Practice Name</label>
+            <input [(ngModel)]="practiceEntry" class="form-control">
           </div>
         </div>
-      </mat-step>
-
-      <mat-step matStepContent label="Gender ">  
-    <div class="StepContainer" >
-    <div class="StepTitle">Who does the practice?</div>
-    <div class="StepContent">
-      <div class="genderDiv"> 
-        <div class="genderType" style="color: #800080;"
-        (click)="handleGender('female')"
-         [ngStyle]="{'background-color': gender.includes('female') ? 'pink' : 'lightgray'}"
-          >Female</div>
-        <div class="genderType" (click)="handleGender('male')" style="color: blue;" 
-        [ngStyle]="{'background-color': gender.includes('male') ? 'pink' : 'lightgray'}" >Male</div>
+        <div>
+          <button [style.opacity]="practiceEntry ? 1 : 0.5" [disabled]="!practiceEntry" style="background-color: #ccc;"
+            mat-button mat-button matStepperNext>Next</button>
+        </div>
       </div>
-      <div style="height: 1rem;"> {{gender.length>1 ? gender.join(" and "): gender[0]}} </div>
-     </div>
-          <div class="ButtonSection">
-            <button 
-            style="background-color: #ccc;"
-            mat-button matStepperPrevious>Back</button>
-            <button 
-            [style.opacity]="(gender.length>0) ? 1 : 0.5" 
-        [disabled]="!(gender.length>0)" 
-            style="background-color: #ccc; margin-left: 2rem;"
-            mat-button matStepperNext>Next</button>
+    </mat-step>
+
+    <mat-step matStepContent label="Gender ">
+      <div class="StepContainer">
+        <div class="StepTitle">Who does the practice?</div>
+        <div class="StepContent">
+          <div class="genderDiv">
+            <div class="genderType" style="color: #800080;" (click)="handleGender('female')"
+              [ngStyle]="{'background-color': gender.includes('female') ? 'pink' : 'lightgray'}">Female 
+              <!-- <img
+                *ngIf="femaleIcon" [src]="femaleIcon" /> -->
```

Comment:
nit(non-blocking)
If the code is no longer needed probably cleaner to just remove, git keeps code history if you ever need to recover. There's a few comments in this file that could probably be deleted